pkg/network/amqp.go

```diff
@@ -58,11 +67,28 @@ func (a *Amqp) Stop() {
 }
 
 // PublishPersistentMessage sends a persistent message to RabbitMQ
-func (a *Amqp) PublishPersistentMessage(exchange, exchangeType, key string, body []byte, headers amqp.Table) error {
-	err := a.declareExchange(exchange, exchangeType)
+func (a *Amqp) PublishPersistentMessage(exchange, exchangeType, key string, msg interface{}, options *MessageOptions) error {
+	var err error
```

Comment:
Is it necessary to declare this variable here?

Reply:
No, removed it!